### PR TITLE
Fix multiple cards not being able to drag into other columns in kanban view

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary

Multiple Cards not being able to be dragged into other columns in kanban view

#### Ticket Link

Fixes GH-719

image

image

### CODE REVIEW

**Comment by hahmadia** [comment]
The problem here was that `selectedCardIds` was coming in as `[]` all the time (due to functional component problem). 

Thanks to Harshil for helping me debug it quick and solving the problem via the use of Ref's.

**Comment by kamre** [comment]
@hahmadia can you also check `useDrop` dependencies for this issue?

File: webapp/src/components/kanban/kanban.tsx [hunk 1]
```diff
@@ -103,19 +105,18 @@ const Kanban = (props: Props) => {
     }
 
     const onDropToColumn = async (option: IPropertyOption, card?: Card, dstOption?: IPropertyOption) => {
```
Comment (jespino) [inline webapp/src/components/kanban/kanban.tsx:107]: I would solve this with a `useCallback`, fix it like this for now.

**Review by jespino — APPROVED** [review]
LGTM

**Comment by hahmadia** [comment]
@jespino @harshilsharma63 

Closing this PR as I was unable to reproduce this problem in Main so it seems to have been resolved.

**Pull request closed without merging**